server/views/pages/event.njk

```diff
@@ -24,58 +24,69 @@
+                    <div>{{ event.format.title }}</div>
+                  </span>
+                  <span>
+                    {% if (event.accessOptions.size !== 0) %}
+                      <div class="{{ {s:'HNL4', m:'HNL5', l:'HNL3', xl:'HNL2'} | fontClasses }}">
+                        {% for accessOption in event.accessOptions.toJS() %}
+                          <div class="flex">
+                            <div class="{{ {s:1} | spacingClasses({margin:['right']}) }}">{% icon 'a11y/' + (accessOption.accessOption.acronym | lower), accessOption.accessOption.title %}</div>
+                          </div>
+                        {% endfor %}
+                      </div>
+                    {% endif %}
                   </span>
                 </div>
 
-                <h1 class="{{ {s:'WB5', m:'WB6', l:'WB4', xl:'WB3'} | fontClasses }} {{ {s: 0} | spacingClasses({margin:['bottom']}) }}">{{ event.title }}</h1>
-                <div>{{ event.format.title }}</div>
 
-                {% if (event.accessOptions.size !== 0) %}
-                  <div class="{{ {s:'HNL4', m:'HNL5', l:'HNL3', xl:'HNL2'} | fontClasses }}">
-                    {% for accessOption in event.accessOptions.toJS() %}
-                      <div class="flex">
-                        <div class="{{ {s:1} | spacingClasses({margin:['right']}) }}">{% icon 'a11y/' + (accessOption.accessOption.acronym | lower), accessOption.accessOption.title %}</div>
-                        <p class="no-margin">{{ accessOption.accessOption.title }}</p>
-                      </div>
-                    {% endfor %}
-                  </div>
-                {% endif %}
+                <h1 class="{{ {s:'WB5', m:'WB6', l:'WB4', xl:'WB3'} | fontClasses }} {{ {s: 0} | spacingClasses({margin:['bottom', 'top']}) }}">{{ event.title }}</h1>
 
+                <div class="flex flex--v-center {{ {s: 2} | spacingClasses({margin: ['top']}) }}">
+                  {% icon 'other/ticket' %}
+                  <strong class="{{ {s:'WB7', l:'WB6'} | fontClasses }} {{ {s: 2} | spacingClasses({margin: ['left']}) }} {{ {s: 6} | spacingClasses({margin: ['right']}) }}">
+                    <span class="caps">Free</span>
+                    {{ ' / Drop in' if event.isDropIn }}
+                  </strong>
+                </div>
 
-                <span class="{{ {s:'HNL4', m:'HNL5', l:'HNL3', xl:'HNL2'} | fontClasses }}">
-                  {% for dateRange in event.when.toJS() %}
-                    {% set formattedDateRange = dateRange.start | formatAndDedupeOnDate(dateRange.end) %}
-                    <div class="{{ {s: 2} | spacingClasses({margin: ['top']}) }}">
+                <span class="flex {{ {s:'HNL4', m:'HNL5', l:'HNL3', xl:'HNL2'} | fontClasses }} {{ {s: 2} | spacingClasses({margin: ['top']}) }}">
+                  <div>
+                    {% icon 'actions/comments' %}
```

Comment:
I'm guessing this is a placeholder for the calendar?

Reply:
I'd really be interested to have the chat  around what we think with the `m` breakpoint and is the user benefit worth the effort of maintaining it, or would the `s` version be a good fit?

Reply:
Before making any decisions on this, we would really need to review this across all the designs we have done. While it worked as a quick fix on the events page we discussed @jamesgorrie I cannot say it would work elsewhere.

Reply:
@Norvard the point of copy / pasting exhibitions was not to do any quick fixes, which I think we've avoided.

Let's deflect minor changes into a bigger piece of holistic design work (font sizes etc that's being spoken about with @Heesoomoon).